saleor/graphql/schema.graphql

```diff
@@ -2253,6 +2256,9 @@ type Mutation {
   productVariantDelete(id: ID!): ProductVariantDelete
   productVariantBulkCreate(product: ID!, variants: [ProductVariantBulkCreateInput]!): ProductVariantBulkCreate
   productVariantBulkDelete(ids: [ID]!): ProductVariantBulkDelete
+  productVariantStocksBulkCreate(stocks: [StockInput!]!, variantId: ID!): ProductVariantStocksBulkCreate
```

Comment:
As far as I see the arguments are generated in alphabetic order. 
What about  adding "bulk" to mutation name, I used it, because this mutations creates more than one stock instance and others mutations, which bahave in this way, have "bulk" in their names. I think that word "stocks" suggests enough that it's creating more than one stock instance, but I wanted to be  consistent.

Reply:
I changed mutations' names.